sway/ipc-json.c

```diff
@@ -85,6 +85,42 @@ static const char *ipc_json_output_transform_description(enum wl_output_transfor
 	return NULL;
 }
 
+#if HAVE_XWAYLAND
+static const char *ipc_json_xwindow_type_description(struct sway_view *view) {
```

Comment:
This needs an alternative implementation for when wlroots is built without Xwayland support.

Reply:
The call to this function is already guarded behind `#if HAVE_XWAYLAND`.

Reply:
So it seems.